lib/kubeclient/common.rb

```diff
@@ -109,6 +109,10 @@ def self.define_entity_methods(entity_types)
         define_method("update_#{entity_name}") do |entity_config|
           update_entity(entity_type, entity_config)
         end
+
+        define_method("patch_#{entity_name}") do |name, namespace = nil, patch|
```

Comment:
@zakiva :+1: to move `namespace` at the end. The consistency for me is "`namespace` is optional, therefore is always at the end", so for me it's consistent to `get` and `delete` as well.